libs/blocks/modal/modal.css

```diff
@@ -49,6 +49,10 @@
   outline: 2px solid var(--modal-focus-color);
 }
 
+.dialog-modal > .fragment > .section > .content {
```

Comment:
I'm thinking the `.dialog-modal > .fragment > .section` shouldn't have such a deep selector which you then need to overwrite with yet another level...

Reply:
Interesting issue we have here because `dialog-modal` and `fragment` are technically the namespaces for these two autoblocks. Do you think I should change the selector in the fragments file to `.fragment > .content` and this one to `.dialog-modal > .content` @mokimo ?

Reply:
If that works, that would seem better yeah!

Reply:
@mokimo I made the specificity a bit better. I decided not to change the fragment styles in `style.css` since there might be edge cases of which I am unaware. Let me know what you think!